src/main/java/step1/domain/StringAddCalculator.java

```diff
@@ -0,0 +1,43 @@
+package step1.domain;
+
+import step1.exception.CharacterAndMinusCustomException;
+import step1.util.StringUtil;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private String value;
+    private static final Pattern COMPARE_NUMERIC = Pattern.compile("-?\\d+(\\.\\d+)?");
+
+
+    public StringAddCalculator(String value) {
+        this.value = value;
+    }
+
+    public int splitAndSum() {
+        if (StringUtil.isEmpty(value)) {
+            return 0;
+        }
+
+        StringSplit stringSplit = new StringSplit(value);
+        String[] numberValue = stringSplit.splitWithDelimeter();
+        return Arrays.stream(numberValue).mapToInt(stringNumber -> stringToInt(stringNumber)).sum();
```

Comment:
개인적으로 메서드 체이닝 사용시 가독성을 위해 한줄에 하나의 dot(.)만 사용하는게 좋다고 생각해요.
```suggestion
        return Arrays.stream(numberValue)
                     .mapToInt(stringNumber -> stringToInt(stringNumber))
                     .sum();
```